pom.xml

```diff
@@ -53,78 +52,27 @@
       <artifactId>mailer</artifactId>
       <version>1.8</version>
     </dependency>
-    <dependency>
-      <groupId>com.google.code.findbugs</groupId>
-      <artifactId>annotations</artifactId>
-      <version>3.0.0</version>
-      <scope>provided</scope>
-    </dependency>
   </dependencies>
 
   <build>
-    <pluginManagement>
-      <plugins>
-        <plugin>
-          <artifactId>maven-release-plugin</artifactId>
-          <version>2.5</version>
-        </plugin>
-      </plugins>
-    </pluginManagement>
     <plugins>
       <plugin>
-        <groupId>org.codehaus.mojo</groupId>
-        <artifactId>animal-sniffer-maven-plugin</artifactId>
-        <!-- details defined in the parent POM -->
-      </plugin>
-      <plugin>
         <groupId>org.codehaus.gmaven</groupId>
         <artifactId>gmaven-plugin</artifactId>
         <executions>
```

Comment:
Yes, I think so. If removed, the version used will be a greater one and the plugin stops building due to an API incompatibility. Specifically, this one:

```
 [ERROR] Failed to execute goal org.codehaus.gmaven:gmaven-plugin:1.5-jenkins-3:testCompile (test-in-groovy) on project ldap: Execution test-in-groovy of goal org.codehaus.gmaven:gmaven-plugin:1.5-jenkins-3:testCompile failed: An API incompatibility was encountered while executing org.codehaus.gmaven:gmaven-plugin:1.5-jenkins-3:testCompile: java.lang.NoSuchMethodError: org.codehaus.groovy.ast.ModuleNode.getStarImports()Ljava/util/List;
```